core/src/main/java/org/elasticsearch/search/suggest/phrase/PhraseSuggestionContext.java

```diff
@@ -30,7 +30,7 @@
 import java.util.List;
 import java.util.Map;
 
-class PhraseSuggestionContext extends SuggestionContext {
+public class PhraseSuggestionContext extends SuggestionContext {
```

Comment:
Nope, not sure why I did that, don't even recall making the change!  I'll change it back.